pkg/api/cached.go

```diff
@@ -287,3 +355,13 @@ func getFolderSize(path string) (int64, error) {
 	})
 	return totalSize, err
 }
+
+func execCommand(cmdName string, args ...string) error {
```

Comment:
In github actions we're not running as root, in our deployment we run as root in a privileged container so this is how we can call `mount`/`umount` during our tests. If there's a more elegant way to do this let me know.